main/tests/cypress/cypress/e2e/import-project/import_project.cy.js

```diff
@@ -47,4 +47,31 @@ describe(__filename, function () {
     cy.waitForProjectTable();
     cy.get('#project-name-button').contains(projectName);
   });
+
+  it('Import a project from URL', function () {
+    cy.visitOpenRefine();
+    cy.navigateTo('Import project');
+    const url = "https://pintoch.ulminfo.fr/6e551ee9a1/Changing-the-type-of-the-column-for-numeric-1680159450732.openrefine.tar.gz";
```

Comment:
This is a URL that points to my personal server, so I'd rather avoid relying on it in the test itself.
In other places in the Cypress tests we simply point to the download URL offered by GitHub. We happen to already have another fixture of OpenRefine project archive that we could likely use directly:
```suggestion
    const url = "https://raw.githubusercontent.com/OpenRefine/OpenRefine/master/main/tests/cypress/cypress/fixtures/food-small-csv.openrefine.tar.zip";
```
This will likely require adapting the assertions afterwards since the project is different.